duckduckgo/iOS
# 17. Subscription Review Fixes

Task/Issue URL: https://app.asana.com/0/1204099484721401/1206757416402738/f

**Description**:
- Copy and layout updates in Subscription > Add device
- Properly update account email when changes in the Add Device page
- Prevent showing the "Subscription Activation in progress" error while fetching subscription info



**Steps to test this PR**:
⚠️   Use alpha app target and a sandbox user - Test on device
1.  Purchase a subscription from Settings 
2. Go to Settings > Manage Subscription > Add to Another Device
3. If you have an email, remove it and come back to this view
4. Tap "Add Email" & Add an email (Don't tap back to settings)
5. Tap the view back button (Add Device)
6. View should refresh and newly added email should appear.

**Slow network test:**
- Enable network link conditioner on iOS and pick 3G
- Open Settings
- Observe that the subscription section takes a moment to load, and space is blank in the meantime.   (This load time will be automatically fixed once we implement entitlements caching)




---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
